### DISCUSSION TITLE
How to convert rnote to pdf

### DISCUSSION THREAD

**Discussion by sakshamred** [flxzt/rnote#1233] [Q&A] [answered]
I was making some notes . But cant find how to convert it to pdf so i can open it in my ipad

**Comment by Doublonmousse** [comment, marked as answer]
See the video here : 
https://github.com/flxzt/rnote/issues/1199#issuecomment-2324694877
(you'd want to export the document, not the selection but the option is next to this one)

**Reply by sakshamred** [reply]
yeah tried it thanks man
